fbw-a380x/src/systems/instruments/src/SDv2/Pages/Video/VideoPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { FSComponent, VNode } from '@microsoft/msfs-sdk';
+import { DestroyableComponent } from 'instruments/src/MsfsAvionicsCommon/DestroyableComponent';
```

Comment:
This should be a relative path too.
`../../MsfsAvionicsCommon/DestroyableComponent` I think.